app/views/admin/conferences/edit.html.haml

```diff
@@ -26,4 +26,7 @@
         = f.input :end_hour, input_html: {size: 2, type: 'number', min: 1, max: 24}
       = f.inputs name: 'Registrations' do
         = f.input :registration_limit, as: :number, in: 0..9999, hint: 'Limit the number of registrations to the conference (0 no limit). Please note that the registration limit doesn\'t apply to speakers of confirmed events (they will still be able to register even if it has been reached). You currently have ' + pluralize(@conference.registrations.count, 'registration')
+      = f.inputs name: 'Booths' do
+        = f.input :booth_limit, as: :number, in: 0..9999,
+        hint: 'Booth limit is the maximum number of booth requets that you can accept for this conference. By setting this number (0 no limit) you can be sure that you are not going to accept more requests than the number of which your conference can support. You currently have ' + pluralize(@conference.booths.where(state: 'accepted').count, 'accepted booth') +'.'
```

Comment:
* of booths that you can accept..
* ...that you are not going to accept more booths than the conference can accommodate...